docs/runbook.md

```diff
@@ -0,0 +1,245 @@
+
+```bash
+docker-compose down
+docker-compose up -d
+```
+
+## Install A Release
+
+Published Docker images are
+[tagged](https://quay.io/repository/team-helium/router?tab=tags)
+as `production` or `staging` as appropriate.
+
+Install:
+
+```bash
+router install production
+```
+
+Restart the service:
+
+```bash
+docker-compose down && docker-compose up -d
+```
+
+## File System Cleanup
+
+Keep `blockchain` because it contains your swarm key.
+
+Keep `router.db` because it contains device cache.
+
+Everything else under `/var/data/` may be purged for reclaiming space on the
+file system.
+
+## Funding
+
+The wallet address may be extracted from a running server instance:
+
+```bash
+router peer addr
+```
+
+Take only the long alphanumeric sequence from the results *without* `/p2p/`
+prefix:
+
+```
+/p2p/LongAhphaNumeric...
+```
+
+Look at your history of Data Credit consumption for the server, and forecast
+your target amount on that trend.  This example uses a nice round `1.0`
+billion DC.
+
+Convert HNT to DC via prevailing exchange rate from the
+[price oracle](https://api.helium.io/v1/oracle/prices/current).
+
+Optionally, get
+[predicted HNT Oracle Prices](https://api.helium.io/v1/oracle/predictions).
+"If no predictions are returned, the current HNT Oracle Price is valid for at
+least 1 hour."
+
+Calculate `$target_dc / $oracle_price / 1000` or as Lisp expression for
+target of 1.0 billion DC with HNT price of 2425000000 resulting in 412:
+
+```lisp
+(let ((current-hnt-price (/ 2425000000 1000))
+      (target-dc (* 1.0 1000 1000 1000)))
+  (floor (/ target-dc current-hnt-price)))
+```
+
+Use the resulting value as the `amount`:
+
+```bash
+helium-wallet burn --amount 412 --payee 123abc... --commit
+```
+
+Use the [Helium Wallet CLI](https://github.com/helium/helium-wallet-rs/)
+or its equivalent.
+
+## Snapshots
+
+Snapshots can be fetched by URL but as of early 2022 are approaching 300MiB.
+
+Fetch its metadata:
+
+```bash
+wget -N https://snapshots.helium.wtf/mainnet/latest-snap.json
+```
+
+The resulting JSON contains `height`, which gets used for file name of
+snapshot to download.
+
+```json
+{
+    "compressed_hash": "SGflG0oH0TTbairR3JCbtXoGJMhl6pYW4dHExmydRIc",
+    "compressed_size": 262995691,
+    "file_hash": "k0Pp00Y2wIeDagvzFMz66RAbD73e2y8fisCibP2Au2M",
+    "file_size": 427285643,
+    "hash": "3MmAnIgap0AMXL8n9eixxvnN8U8Qwtc3X_N6XXY1oR0",
+    "height": 1279441
+}
+```
+
+Craft the new URL based upon that height:
+
+```bash
+wget -N https://snapshots.helium.wtf/mainnet/snap-1279441.gz
+```
+
+However, those are **generated** from the chain approximately **every 11 hours**.
+
+Therefore, compare chain heights or file timestamps.
+
+```bash
+curl https://api.helium.io/v1/blocks/height
+```
+
+Or:
+
+```bash
+curl --head https://snapshots.helium.wtf/mainnet/snap-1279441.gz
+TZ=Z date
+```
+
+Scripted version:
+
+```bash
+wget -N \
+  $(curl https://snapshots.helium.wtf/mainnet/latest-snap.json | \
+    jq .height | \
```

Comment:
Do any of our servers have `jq` installed?

Reply:
This bit of script is more for quick copy/paste from laptop/workstation in a pinch; e.g, 3am and half asleep.

That said: if such granularity seems excessive for this document, we can remove it.